webpack.config.js

```diff
@@ -5,18 +5,19 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
+const packageJSON = require("./package.json");
+
 const execSync = require('sync-exec'),
       now = new Date(),
       buildDate = `${now.getUTCFullYear()}-${now.getUTCMonth() + 1}-${now.getUTCDate()}`,
       gitBranch = execSync('git symbolic-ref --short HEAD').stdout.trim(),
-      gitTags   = execSync('git describe --tags').stdout,
       gitLog = execSync('git log -1 --date=short --pretty=format:"%cd %h %ce"').stdout;
 
 const envMap = { production: "production", master: "staging" },
       environment = process.env.ENVIRONMENT || envMap[gitBranch] || "development",
       buildInfo = {
         date: buildDate,
-        tag: gitTags.split('-')[0],
+        tag: packageJSON.version,
```

Comment:
Hmn. I am wondering about this one.

git tags are a normal way we manage release versions.  With this change we would need to release using npm / `package.json`.

We don't need to address this now, bu we should revisit.

Reply:
We can add it back but I did it so that the version displayed on the branch build.